src/utils/helpers/loginCheck.ts

```diff
@@ -1,6 +1,6 @@
 const isLoggedIn = () => {
   const JWT = localStorage.getItem('persistJWT')
-  if (JWT == 'undefined') return false
+  if (JWT == undefined) return false
```

Comment:
Pretty sure this needs to be how it was. That was a bug that took ages to track down -- turns out that the local storage can't store "undefined" as a value, so it stores the string "undefined".

Reply:
d140f22a8e2bbdbd6c64ac045290d1b1942610e8

Reply:
You changed the one in `UserState` -- also needs changing in `loginCheck.ts`